src/howard/howard/issuers/certificate.py

```diff
@@ -15,8 +14,8 @@ class Organization(BaseModel):
 
     name: str
     representative: str
-    signature: Path
-    logo: Path
+    signature: str
+    logo: str
```

Comment:
What about being permissive on this?

```suggestion
    signature: Union[Path, str]
    logo: Union[Path, str]
```

Reply:
I used a `constr` with a simple regexp to determine if the provided value looks a base64 image.